src/kvirc/sparser/KviIrcServerParser_literalHandlers.cpp

```diff
@@ -873,8 +873,8 @@ void KviIrcServerParser::parseLiteralPrivmsg(KviIrcMessage * msg)
 
 	PrivmsgIdentifyMsgCapState eCapState = IdentifyMsgCapNotUsed;
 
-	KviCString pTrailing = msg->trailingString();
-	if(pTrailing)
+	KviCString pTrailing = msg->safeTrailingString();
+	if(!strcmp(pTrailing,""))
```

Comment:
Why not `pTrailing.isEmpty()`?

Reply:
I'm the author of this fix. I worked with dem-ionz on the results his fuzzing found.

The simple reason I used strcmp over .isEmpty() is that I guessed it'd automatically convert to a C string and didn't put any time into researching your code base to find methods which could accomplish the same thing.

There's probably no reason not to use .isEmpty() instead.